content/en/platform/corda/5.0/application-networks/creating/members/cpi.md

```diff
@@ -14,9 +14,9 @@ section_menu: corda5
 
 This section describes how to build a member CPI and upload it to the network. It contains the following:
 1. [Set Variables]({{< relref "#set-variables" >}})
-2. [Generate the Group Policy File]({{< relref "#generate-the-group-policy-file" >}})
-3. [Create the CPI File]({{< relref "#create-the-cpi-file" >}})
-4. [Import Code Signing Certificates]({{< relref "#import-code-signing-certificates" >}})
+2. [Create the CPI File]({{< relref "#create-the-cpi-file" >}})
+3. [Import Code Signing Certificates]({{< relref "#import-code-signing-certificates" >}})
+4. [Generate the Group Policy File]({{< relref "#generate-the-group-policy-file" >}})
```

Comment:
Not sure about this order? Is the Group Policy File not required to create the CPI?